src/routes/docs/layouts/tabview.md

```diff
@@ -29,3 +29,5 @@ button_2.grid(margin=20)
 app.run()
 
 ```
+
+The first two blocks are fairly typical for a `Tukaan` program with the necessary import statement and app/window creation. The next block is where the `TabView` is clearly made and is also fairly standard. The next block, however, is slightly unorthodox. In this block we do the following: `tab_x = tabview.Tab("Tab x")`. Digging into the `TabView` source code we find [`self.Tab = Tab`](https://github.com/tukaan/tukaan/blob/f39bb197b1bbbb05a03e498114cc1c4dcff8b800/tukaan/widgets/tabview.py#L152-L155). If we go up in the file we then find class `Tab` which is really just a wrapper around the `Frame` class with extra methods for tabs. Because of how `Tab` is implemented, however, it creates a very clean API call that doesn't require much extra thought to understand. Moving on to the next two blocks we see that these tabs are interacted with exactly as one would work with a `Frame` which of course makes sense knowing that they are indeed a Frame on the inside. The final block is also fairly generic with the call of the `App.run()` method.
```

Comment:
- I don't think it's necessary t explain every part of the code. At this point the reader is expected to know the parts where you use "fairly standard/generic", and I don't want the docs to re-explain these basic patterns of Tukaan everywhere.

- Please don't explain what the source code does, it's irrelevant. All the user should care about here is how they does things with the public API, the rest is implementation detail.

I think what this part should be about is the you can create a new tab by creating a `.Tab()`, and it works like an other container. The user can set parameters of the tab, like the title and icon, and can call different methods on it, like `.move()`, `.hide()`, or `.remove()`

Reply:
Got it 👍